src/main/java/org/jabref/gui/preferences/XmpPrivacyItemModel.java

```diff
@@ -0,0 +1,29 @@
+package org.jabref.gui.preferences;
+
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
+
+import org.jabref.model.entry.field.Field;
+
+public class XmpPrivacyItemModel {
```

Comment:
As this class only contains the `field`, I guess we can remove it and work with the Field directly. (using `BindingsHelper.constantOf` below in the cell factory).